File: js/problem-1010.js
// https://www.beecrowd.com.br/judge/en/problems/view/1010

let input = require('fs').readFileSync('/dev/stdin', 'utf8'),
[product1, product2] = input.split('\n'),
[code1, units1, unitPrice1] = product1.split(' ').map(value => Number(value)),
[code2, units2, unitPrice2] = product2.split(' ').map(value => Number(value)),
price = units1 * unitPrice1 + units2 * unitPrice2;
console.log(`VALOR A PAGAR: R$ ${price.toFixed(2)}`)

File: python/problem-1010.py
# https://www.beecrowd.com.br/judge/en/problems/view/1010

code1, units1, unit_price1 = input().split()
code2, units2, unit_price2 = input().split()
price = float(unit_price1) * int(units1) + float(unit_price2) + int(units2)
print(f"VALOR A PAGAR: R$ {price:.2f}")